helpdesk/views/helpdesk_ticket_view.xml

```diff
@@ -75,11 +75,13 @@
                     </div>
                     <group name="main">
                         <group>
-                            <field name="team_id"/>
-                            <field name="user_id"/>
+                            <field name="team_id" options='{"always_reload": True}'/>
+                            <field name="user_ids" invisible="1" />
+                            <field name="user_id" domain="[('id', 'in', user_ids)]" options='{"always_reload": True}'/>
```

Comment:
Teams may not always be used, it is ok to only allow user_id to be one of the team but it must not force you to pick a team.